cognite/powerops/client/powerops_client.py

```diff
@@ -27,39 +28,47 @@ def __init__(
         self,
         read_dataset: str,
         write_dataset: str,
-        cogshop_version: str,
         config: ClientConfig,
         monitor_dataset: str | None = None,
-        shop_as_a_service: bool = False,
     ):
         self.cdf = CogniteClient(config)
-        self.cogshop_version = cogshop_version
         self.datasets = DataSetsAPI(self.cdf, read_dataset, write_dataset, monitor_dataset)
         self.cog_shop1 = CogShop1Client(self.cdf)
         self.assets = PowerAssetAPI(self.cdf)
         self.afrr_bid = AFRRBidAPI(self.cdf)
         self.day_ahead_bid = DayAheadBidAPI(self.cdf)
-        self.shop = SHOPRunAPI(self.cdf, self.datasets.write_dataset_id, cogshop_version, shop_as_a_service)
-        self.workflow = DayaheadTriggerAPI(self.cdf, self.datasets.write_dataset_id, cogshop_version)
+        print(
+            "The generated modules `cog_shop1`, `assets`, `afrr_bid`, and `day_ahead_bid`"
+            " are deprecated and will be removed."
+        )
+        self.shop = SHOPRunAPI(self.cdf, self.datasets.write_dataset_id)
+        print("Module `shop` (SHOPRunAPI) is deprecated and will be removed.")
+        self.workflow = DayaheadTriggerAPI(self.cdf, self.datasets.write_dataset_id)
+        print("Module `workflow` (DayaheadTriggerAPI) is deprecated and will be removed.")

```

Comment:
**suggestion** (as mentioned on slack): Print deprecation messages only when these are used (if it's not too complicated)